iceoryx2/src/service/attribute.rs

```diff
@@ -164,14 +175,14 @@ impl AttributeSpecifier {
 /// is opened.
 #[derive(Debug)]
 pub struct AttributeVerifier {
-    attribute_set: AttributeSet,
+    required_attributes: AttributeSet,
```

Comment:
@elfenpiff I renamed some things, please check it's OK with you. I find these more explicit names easier to follow in the code ...